packages/web/src/pages/sign-up-page/styles/CreatePasswordPage.module.css

```diff
@@ -1,8 +1,15 @@
-.contentContainer {
-  background-color: var(--static-white);
-  text-align: left;
-}
-
 .formOuterContainer {
   flex-grow: 1;
 }
+
+.backIcon {
+  color: var(--harmony-n-400);
```

Comment:
I think the harmony icons now support a color prop

Reply:
:o amazing